distribution/pom.xml

```diff
@@ -83,6 +74,7 @@
 				<executions>
 					<execution>
 						<id>attach-javadoc</id>
+						<phase>install</phase>
```

Comment:
The caching was supposed to be for dependencies.

http://docs.travis-ci.com/user/caching/

It turns out there needs to be one more line in the travis config (sudo:
false) to make it use the container based infrastructure.

http://docs.travis-ci.com/user/workers/container-based-infrastructure/

I'm doing stuff at Protégé that needs fixes in 3.5.2, so I am doing a lot
of dependency fun and games with locally installed snapshots (including
discovering that reasoner versions can't have non digits. And that you
can't have a SNAPSHOT of a beta-16 and expect maven to work.